e2e-cypress/files/cypress-acceptance.config.ts

```diff
@@ -2,18 +2,30 @@ import { defineConfig } from 'cypress'
 import setupNodeEvents from './plugins/index.js'
 export default defineConfig({
   //projectId: '[Your project ID from Cypress cloud]',
-  reporter: 'reporters/custom-reporter.js',
+  reporter: 'cypress-multi-reporters',
   reporterOptions: {
-    mochaFile: 'build/test-results/acceptance-junit-[hash].xml',
-    toConsole: true,
+    reporterEnabled: 'mochawesome,./reporters/custom-reporter.js',
+    mochawesomeReporterOptions: {
+      reportDir: 'build/test-results/mochawesome',
+      reportFilename: 'acceptance-mochawesome',
+      charts: true,
+      html: true,
+      timestamp: true,
+      json: true
+    },
+    reportersCustomReporterJsReporterOptions: {
+      mochaFile: 'build/test-results/acceptance-junit-[hash].xml',
+      toConsole: true,
+    },
   },
   e2e: {
-    baseUrl: 'https://www.w3schools.com',
+    baseUrl: process.env.CYPRESS_BASE_URL,
```

Comment:
You are right. I'll change the env variable name to BASE_URL so that it makes more sense. I prefer to keep the baseUrl parameter there as I think it's clearer for local env